src/modules/fancyzones/FancyZonesLib/trace.cpp

```diff
@@ -15,6 +15,9 @@
 #define EventSettingsKey "FancyZones_Settings"
 #define EventDesktopChangedKey "FancyZones_VirtualDesktopChanged"
 #define EventZoneWindowKeyUpKey "FancyZones_ZoneWindowKeyUp"
+#define EventMoveNewWindowIntoZone "FancyZones_MoveNewWindowIntoZone"
+#define EventKeyboardSnapWindowToZone "FancyZones_KeyboardSnapWindowToZone"
+#define EventMoveSizeEnterKey "FancyZones_MoveSizeEnter"
```

Comment:
The name is confusing. Maybe we should name it something like `FancyZones_StartWindowDragging`

Reply:
The reason for picking this name is that it's paired with _MoveSizeEnd, which is already out in the world.

Reply:
Perhaps `FancyZones_MoveSizeBegin`? What do you think @mykhailopylyp ?

Reply:
I think we should rename `_MoveSizeEnd`. Any idea why it was called this in the first place? Is it really an event for window dragging?

Reply:
>Perhaps `FancyZones_MoveSizeBegin`

Still, it's not clear what this event is about

Reply:
I think there are pipelines already being built with "MoveSizeEnd" in mind, so it would not be good to rename it. @dedavis6797 , any thoughts on this?

Reply:
I think it should better be `MoveSizeStart` as it is in the event constants. 
https://docs.microsoft.com/en-us/windows/win32/winauto/event-constants

But IMO something like `MoveOrResizeStarted` would be more clear.

Reply:
The data is currently flushed after 30 days (except certain exceptions where we have extensions in place) so I'm inclined to just rename the existing event to add clarity. `FancyZones_MoveWindowIntoZoneStarted` and `FancyZones_MoveWindowIntoZoneEnded` seem reasonable.